### PR TITLE
texworks: init at 0.6.2

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/editors/texworks/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ qt5.qtscript libsForQt5.poppler hunspell lua python ]
+                ++ lib.optional withLua lua
+                ++ lib.optional withPython python;
+
+  cmakeFlags = lib.optional withLua "-DWITH_LUA=ON"
+               ++ lib.optional withPython "-DWITH_PYTHON=ON";
+
+  meta = with stdenv.lib; {
+    description = "simple TeX front-end program inspired by TeXShop";
```
Comment (vyp) [inline pkgs/applications/editors/texworks/default.nix]: First letter of description should be capital https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

Comment (dotlambda) [reply pkgs/applications/editors/texworks/default.nix]: Corrected that

**Comment by dotlambda** [comment]
Before you merge this, I'd like to know what the policy is regarding the `platforms` meta attribute. The build process on macOS seems to be the same as on Linux, but I have no system to test this on.
Would it still be okay to set `meta.platforms` to `platforms.unix`?

**Comment by Mic92** [comment]
I would say it is better to set it to linux first and wait for somebody to test it on macos. It might not even build for some reason due different dependencies.

**Comment by dotlambda** [comment]
Okay, you can merge it now

**Comment by Mic92** [comment]
Thanks!

**Pull request merged**